app/controllers/application_controller.rb

```diff
@@ -9,6 +9,8 @@ class ApplicationController < ActionController::Base
 
   include ApplicationHelper
 
+  rescue_from InvalidRequest, with: :render_error_response
```

Comment:
How can one trigger this error? Just curious

Reply:
so -- i was look at using exceptions over the nested if's in a lot of the controllers with early returns.  I created this custom exception in the initializers, and was throwing it in a test branch -- looks like it slipped in.  Can remove.

Reply:
Let's just keep it. Its always better than a Internal Server Error